### ISSUE TITLE
[PdrDashboard, v0.2] Objectify data results and manipulation

### ISSUE THREAD

**Issue by calina-c** [oceanprotocol/pdr-backend#1373]
This is better. But I think we can do even better. Instead of displaying "table data" let's display objects directly. We can add a wrapper to the queries that converts table data into objects and we can work with them much more easily. 

contracts is a column to object mapping, that takes table data and makes Contract objects
arg_pairs is a column pair to object mapping, that takes table data and makes ArgPair objects.

long story short, let's separate concerns for list creation and list display. Move out list creation from callbacks.py because it is not its concern.

_Originally posted by @calina-c in https://github.com/oceanprotocol/pdr-backend/pull/1326#discussion_r1671796965_
